src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -55,6 +97,10 @@ public class DependencyGraph
         /// </summary>
         public IEnumerable<DependencyGraphNode> IncoherentNodes { get; set; }
 
+        public IEnumerable<Build> ContributingBuilds { get; set; }
+
+        public IEnumerable<DependencyDetail> DependenciesMissingBuilds { get; set; }
```

Comment:
What's a "DependenciesMissingBuilds"? The other stuff has fairly detailed comments.